src/obu.rs

```diff
@@ -116,8 +116,55 @@ use std::cmp;
 use std::ffi::c_int;
 use std::ffi::c_uint;
 use std::ffi::c_void;
+use std::fmt;
 use std::mem::MaybeUninit;
 
+struct Debug {
+    enabled: bool,
+    name: &'static str,
+    init_ptr: *const u8,
+}
+
+impl Debug {
+    pub const fn new(enabled: bool, name: &'static str, gb: &GetBits) -> Self {
```

Comment:
Why is this const?

Reply:
Since the `enabled: bool` was previously a `const` as `const DEBUG_*: bool = false`, so I wanted to keep the initialization `const`.